paimon-spark/paimon-spark-ut/src/test/scala/org/apache/paimon/spark/sql/DDLWithHiveCatalogTestBase.scala

```diff
@@ -134,6 +134,69 @@ abstract class DDLWithHiveCatalogTestBase extends PaimonHiveTestBase {
     }
   }
 
+  test(
+    "Paimon partition expire test with hive catalog: expire partition for paimon table sparkCatalogName") {
+    Seq(paimonHiveCatalogName).foreach {
+      catalogName =>
+        spark.sql(s"USE $catalogName")
```

Comment:
We should focus on what the case itself requires, without introducing other elements. For example, if we are testing a catalogName, there is no need for a foreach, nor do we need variables like comment, location, etc.

Reply:
Done